### ISSUE TITLE
Anyone able to reproduce the results of AnySplat checkpoint from training or finetuning?

### ISSUE THREAD

**Issue by ricshaw** [OpenRobotLab/AnySplat#102]
Hi,

I am having issues reproducing the quality of the provided AnySplat checkpoint using the given code.
I have tried training from scratch (VGGT weights) and also starting from the pretrained AnySplat model.
If I start from pretrained AnySplat checkpoint, the initial PSNR is around 24 dB (good!), but then just drops immediately....

<img width="193" height="142" alt="Image" src="https://github.com/user-attachments/assets/e14f800f-e26a-48c9-9f89-4f8399a64426" />

Here is the result at step 0:

<img width="470" height="238" alt="Image" src="https://github.com/user-attachments/assets/611e68be-f8b7-4613-8c8b-dde9c4d13b8f" />

But then at step 1000:

<img width="473" height="239" alt="Image" src="https://github.com/user-attachments/assets/64e819e4-cbf7-4c91-a1cc-28c29bc7e122" />

And at 30,000 it's still not as good the original checkpoint:

<img width="470" height="239" alt="Image" src="https://github.com/user-attachments/assets/2f8106fa-9fa0-4463-b2d9-4457108eb00b" />

**Comment by myc634** [comment]
Hi, please check out your code for dataset(alignment/pose etc...), we never encountered this problem during our training

**Comment by ricshaw** [comment]
Regarding dataset alignment/pose, I am not using any ground truth poses or depth maps, only inputs images which get center cropped. So extrinsics and intrinsics from the dataset are not used.  I have set normalize_by_pts3d: false and  rescale_to_1cube: false. The only supervision of extrinsics/intrinsics/depth comes from the VGGT distillation network. 
Am I missing something?

**Comment by myc634** [comment]
that should be okay, what batch size you are using? and what is the view sample strategy?

**Comment by ricshaw** [comment]
I'm using the dynamic batch sampler, which has max 24 images. Using bounded view sampler

**Comment by ricshaw** [comment]
Did you train with any normal loss or normal consistency losses? I see weight_normal: 0.0 in the configs.
Also, do you use any masks when computing losses? I am not loading any valid_masks in my dataloader

**Comment by ricshaw** [comment]
Are these the correct params you used for the final checkpoint? 
train:
  pose_loss_alpha: 1.0
  pose_loss_delta: 1.0
  cxt_depth_weight: 0.0
  weight_pose: 10.0
  weight_depth: 1.0
  weight_normal: 0.0

in multi-dataset.yaml you have weight_depth: 0.0
